hazelcast/src/main/java/com/hazelcast/map/impl/MapKeyLoader.java

```diff
@@ -250,10 +252,12 @@ private void sendBatch(Map<Integer, List<Data>> batch, boolean replaceExistingVa
             futures.add(opService.invokeOnPartition(SERVICE_NAME, op, partitionId));
         }
 
-        LoadAllOperation op = new LoadAllOperation(mapName, Collections.<Data>emptyList(), replaceExistingValues, lastBatch);
-
         // notify SENDER_BACKUP
-        futures.add(opService.createInvocationBuilder(SERVICE_NAME, op, mapNamePartition).setReplicaIndex(1).invoke());
+        InternalPartition partition = partitionService.getPartition(mapNamePartition);
+        if (hasBackup && partition.getReplicaAddress(1) != null) {
```

Comment:
The fix will solve the original problem which was unnecessary warning messages in the logs due to sending to non-existing replica.
Need to have another look how it affects failover...